### PR TITLE
[1822PNW] Don't allow tile placement through blue and brown hexes

### PR DESCRIPTION
**Pull request by outsidePasser**

### CODE REVIEW

File: lib/engine/game/g_1822_pnw/game.rb [hunk 1]
```diff
@@ -82,7 +82,7 @@ def price_movement_chart
         PRIVATE_CLOSE_AFTER_PASS = %w[P11].freeze
         PRIVATE_PHASE_REVENUE = %w[].freeze # Stub for 1822 specific code
 
-        IMPASSABLE_HEX_COLORS = %i[gray red].freeze
+        IMPASSABLE_HEX_COLORS = %i[gray red blue brown].freeze
```
Comment (crericha) [inline lib/engine/game/g_1822_pnw/game.rb:85]: Is this compatible with the bridge builder and ports?

Comment (outsidePasser) [reply lib/engine/game/g_1822_pnw/game.rb:85]: Yes.  Currently IMPASSABLE_HEX_COLORS is only used in connect_hexes and lay_tile to set up neighbors, it's not used as a restriction to placing a tile.

**Comment by outsidePasser** [comment]
DO NOT MERGE IN YET - one of these changes caused a problem that I'm still attempting to identify.

**Comment by outsidePasser** [comment]
Ok to merge this in now

**Pull request merged**